components/LinkCustom/LinkCustom.tsx

```diff
@@ -0,0 +1,36 @@
+// Import necessary modules from Next.js and React
+import Link from 'next/link';
+import React from 'react';
+
+interface LinkCustomProps {
```

Comment:
M: Can you change this to `ILinkCustomProps` to match the same naming conventions for the other interfaces in the application?

Reply:
Where else is this used?

Reply:
In https://github.com/LetsGetTechnical/gridiron-survivor/blob/develop/api/IapiFunctions.ts and https://github.com/LetsGetTechnical/gridiron-survivor/blob/develop/store/dataStore.ts, interfaces follow this naming convention

Reply:
Huh, I have _never_ seen a naming convention like that. I don't like it, but I'll update it for consistency. Thanks for pointing this out.